core/domain/rules_registry_test.py

```diff
@@ -121,6 +123,6 @@ def test_get_html_field_types_to_rule_specs_for_given_state_schema_version_with_
             expected_state_schema_version_to_html_field_types_to_rule_specs
         )
 
-        rules_registry.Registry._state_schema_version_to_html_field_types_to_rule_specs[41] = (  # pylint: disable=protected-access
-            specs_from_json_v41
-        )
+        rules_registry.Registry._state_schema_version_to_html_field_types_to_rule_specs[ # pylint: disable=protected-access
+                41
```

Comment:
Deindent this and the following line by 4, so that the closing ']' matches the indentation scope of the opening '['.